### PR TITLE
Motion comments fix and close open dialogs if a relogin is necessary

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
No changelog entry required (bugfixing).

**Comment by emanuelschuetze** [comment]
@normanjaeckel please review later. thx.

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -987,7 +987,7 @@ angular.module('OpenSlidesApp.motions.site', [
             function (motion) {return motion.category ? motion.category.name : '';},
             function (motion) {return motion.motionBlock ? motion.motionBlock.name : '';},
             function (motion) {return motion.recommendation ? motion.getRecommendationName() : '';},
-            function (motion) {return motion.comments.join(' ');},
+            function (motion) {return _.filter(motion.comments, function (comment) {return !!comment;}).join(' ');},
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: Please just write `function (motion) {return _.filter(motion.comments).join(' ');},`

File: openslides/users/static/js/users/site.js [hunk 2]
```diff
@@ -109,6 +109,8 @@ angular.module('OpenSlidesApp.users.site', [
     }
 ])
 
+.value('LoginDialog', {id: undefined})
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: You can use an empty object here.

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Please look at the two remarks.

File: openslides/motions/static/js/motions/site.js (hunk 1 above)
Comment (FinnStutzenstein) [reply openslides/motions/static/js/motions/site.js]: Done.

File: openslides/users/static/js/users/site.js (hunk 2 above)
Comment (FinnStutzenstein) [reply openslides/users/static/js/users/site.js]: Done.

**Pull request merged**